lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -681,6 +683,26 @@ func (c *ContainerProxy) StreamContainerStats(ctx context.Context, config *conve
 	return nil
 }
 
+func (c *ContainerProxy) GetContainerChanges(vc *viccontainer.VicContainer) (io.ReadCloser, error) {
```

Comment:
Need to have a method to export container r/w layer. The only difference with this method is to set data parameter to true while calling the ArchiveExportReader.
So wondering is it better to add one more parameter in this method?